dev-lang/vala/vala-0.36.8.recipe

```diff
@@ -0,0 +1,133 @@
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2016 Jürg Billeter
+	2006-2016 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="df51128ce6ef2505a203c4f17e8269f0edcb5f84619ed562d25a88c52652e5db"
+SOURCE_DIR="vala-$portVersion"
+PATCHES="vala-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	vala$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:vala$secondaryArchSuffix
+	cmd:valac$secondaryArchSuffix
+	cmd:vapicheck$secondaryArchSuffix
+	cmd:vapigen$secondaryArchSuffix
+	cmd:vala_gen_introspect$secondaryArchSuffix
+	cmd:vala_${portVersion%.*}$secondaryArchSuffix
+	cmd:vala_gen_introspect_${portVersion%.*}$secondaryArchSuffix
+	cmd:valac_${portVersion%.*}$secondaryArchSuffix
+	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
+	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	glib2${secondaryArchSuffix} >= 2.32.0
+	vala${secondaryArchSuffix}_common == $portVersion
+	lib:libdbus_1$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	"
+PROVIDES_common="
+	vala${secondaryArchSuffix}_common = $portVersion
+	"
+REQUIRES_common="
+	haiku$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	vala${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	#lib:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	vala$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	glib2${secondaryArchSuffix}_devel >= 2.32.0
+	devel:libdbus_1$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:bison
+	cmd:cmp
+	cmd:diff
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libvala$secondaryArchSuffix \
+	$libDir/libvala-${portVersion%.*}.so.0.0.0 \
+	$binDir/vala \
+	$binDir/valac \
+	$binDir/vapicheck \
+	$binDir/vapigen \
+	$binDir/vala-${portVersion%.*} \
+	$binDir/valac-${portVersion%.*} \
+	$binDir/vapicheck-${portVersion%.*} \
+	$binDir/vapigen-${portVersion%.*}
+	# looks like it doesn't take more then one "-" here
+	#$binDir/vala-gen-introspect \
+	#$binDir/vala-gen-introspect-${portVersion%.*} \
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f $libDir/*.la
+
+	# for some reason it can't install the library
+	prepareInstalledDevelLibs #libvala-${portVersion%.*}
+	fixPkgconfig
+
+	if [ -n "$secondaryArchSuffix" ]; then
+		rm -rf $manDir
+		maybe_manDir=
+	else
+		maybe_manDir=$manDir
+	fi
```

Comment:
As it is now, there are no manpages installed, can't use the x86_gcc2 arch as that isn't supported, any hints here?